three-data-centers-in-two-cities-deployment.md

```diff
@@ -175,10 +175,15 @@ In the deployment of three AZs in two regions, to optimize performance, you need
 - Optimize the network configuration of the TiKV node in another region (San Francisco). Modify the following TiKV parameters for AZ3 in San Francisco and try to prevent the replica in this TiKV node from participating in the Raft election.
 
     ```yaml
-    raftstore.raft-min-election-timeout-ticks: 1000
-    raftstore.raft-max-election-timeout-ticks: 1200
+    raftstore.raft-min-election-timeout-ticks: 50
+    raftstore.raft-max-election-timeout-ticks: 60
     ```
 
+> Note:
+>
+> Setting larger values for `raftstore.raft-min-election-timeout-ticks` and `raftstore.raft-max-election-timeout-ticks` can significantly decrease the likelihood of peers on the target TiKV instance becoming region leaders. However, in a disaster scenario where some TiKV instances are offline and the peers on other instances' Raft logs have fallen behind, it is possible that only the peer on the affected TiKV instance can become a region leader. This peer requires at least raftstore.raft-min-election-timeout-ticks seconds to start a campaign. Therefore, users should avoid setting excessively high configuration values to prevent impacting the cluster's availability in this extreme scenario.
+
```

Comment:
```suggestion
> Configuring larger values for `raftstore.raft-min-election-timeout-ticks` and `raftstore.raft-max-election-timeout-ticks` can greatly reduce the probability that a peer on this TiKV will become the leader. However, in a disaster scenario, if some TiKV nodes are down and the active TiKV node is lagging behind in Raft logs, only the Region on this TiKV can become the leader. When the leader is down, the Region on this TiKV node needs to wait for at least the time set by `raftstore.raft-min-election-timeout-ticks' to become the leader. It is recommended that you do not set these values too large to avoid affecting the availability of the cluster in this scenario.
```